pyannote/audio/applications/change_detection.py

```diff
@@ -318,11 +335,19 @@ def main():
 
         purity = float(arguments['--purity'])
 
+        metric_type = arguments['--metric_type']
+        if not metric_type:
+            metric_type = SEGMENTATION_METRIC_TYPE
```

Comment:
I believe it should be `DIARIZATION_METRIC_TYPE` in order not to change the default behavior.

Reply:
But this won't be needed if you change the application docstring as suggested above.
`docopt` will automagically set it to `"diarization"` thanks to `[default: diarization]`.

Reply:
I would argue that it's better to have the "segmentation" metric as the default one, what do you think?

Reply:
Thinking again about this, I now agree with you. "segmentation" it is...
Still, please use the docopt way of setting the default value for `--metric`.